ci/docker_sources/hil_container/Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM centos
+
+RUN yum install -y epel-release
+
+RUN yum -y update && yum install -y bridge-utils  gcc  httpd  ipmitool libvirt libxml2-devel  libxslt-devel  mod_wsgi net-tools python-pip python-psycopg2 python-virtinst qemu-kvm telnet vconfig virt-install
+
+RUN yum install -y git
+
+RUN yum install -y python-devel
+
+RUN pip install dumb-init
+
+RUN useradd -ms /bin/bash hil
+
+RUN passwd -d hil
+
+RUN passwd -d root
+
+RUN usermod -aG wheel hil
+
+WORKDIR /home/hil
+
+RUN git clone https://github.com/CCI-MOC/hil.git
+
+WORKDIR /home/hil/hil
+
+RUN git reset --hard f88d2fc92066c1e83e428ebeeca351971f306053
```

Comment:
We can directly clone the tagged version instead of doing a `git reset --hard`
`git clone https://github.com/cci-moc/hil -b v0.3`